### ISSUE TITLE
Election Title Overflow - Incorrect Display - Election Title and Description - test maximum length

### ISSUE THREAD

**Issue by masiarek** [Equal-Vote/star-server#485]
Both fields Overflow:
- Title - max 130 characters
- Description (999 characters)
<img width="992" alt="image" src="https://github.com/Equal-Vote/star-server/assets/857777/59844e79-974e-4b6b-b3d8-05da7542ed41">

**Comment by ArendPeter** [comment]
I'm pretty sure this would have wrapped if there were spaces. This usecase of long words is pretty niche, but it should be a easy fix

Looks like overflow-wrap: break-word is what we need

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_text/Wrapping_breaking_text

**Comment by ArendPeter** [comment]
I'm not sure if we fixed this, it needs to be re-tested and then fixed if it's not already

**Comment by mickaelfo** [comment]
> I'm not sure if we fixed this, it needs to be re-tested and then fixed if it's not already

![image](https://github.com/user-attachments/assets/30612fb9-44d7-4bd0-b5a9-6cfe868b6919)

**Comment by masiarek** [comment]
the way I tested is a bit unusual (**without** spaces) - normally we should have spaces.
Not sure if fixing the issue without the spaces is worth the effort.

As long as long text wraps around **with** spaces as OK - we are good.

**Comment by masiarek** [comment]
It looks with spaces (hence issue can be probably closed - if we ignore strange case without spaces):
<img width="673" alt="image" src="https://github.com/user-attachments/assets/f0c22da8-b86e-462b-a82d-9a96905a37c5" />

**Comment by mickaelfo** [comment]
Yep, with spaces it looks okay. I think this issue can be closed.

**Comment by ArendPeter** [comment]
I'll leave it open. It's not common, but the site looks really broken when it happens so I think it's worth fixing.
